Marlin/planner.cpp

```diff
@@ -741,29 +703,17 @@ void Planner::_buffer_line(const float &a, const float &b, const float &c, const
   //*/
 
   // DRYRUN ignores all temperature constraints and assures that the extruder is instantly satisfied
-  if (DEBUGGING(DRYRUN)) {
+  if (DEBUGGING(DRYRUN))
     position[E_AXIS] = target[E_AXIS];
-    #if ENABLED(LIN_ADVANCE)
-      position_float[E_AXIS] = e;
-    #endif
-  }
 
   long de = target[E_AXIS] - position[E_AXIS];
 
-  #if ENABLED(LIN_ADVANCE)
-    float de_float = e - position_float[E_AXIS]; // Should this include e_factor?
```

Comment:
@Sebastianv650 — The same thing applies here. `de_float` is the same as `de` (within a microstep). to get `de_float` from `de` just multiply by `steps_to_mm[E_AXIS_N]`. So, there's no special reason to use `position_float`.

I'm at a loss to perceive any difference between the changed code and the version with `position_float`. I wouldn't be surprised if restoring `position_float` failed to resolve the issue. But, we can try it!